.github/CODEOWNERS

```
@@ -78,16 +90,16 @@ sycl/source/half_type.cpp @AlexeySachkov
 sycl/include/CL/sycl/swizzles.def @turinevgeny
 sycl/include/CL/sycl/types.hpp @turinevgeny
 
-sycl/plugins/cuda/ @intel/llvm-reviewers-cuda
-
-sycl/doc/ @pvchupin @kbobrovs
-
-sycl/doc/extensions/ @intel/dpcpp-specification-reviewers
-
+# XPTI instrumentation utilities
 xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
 
+# DPC++ tools
+llvm/tools/file-table-tform/ @kbobrovs @AlexeySachkov
+llvm/tools/llvm-foreach/ @Fznamznon @AlexeySachkov
```

I suggest @AlexeySachkov as primary code owner for this tool.
```suggestion
llvm/tools/llvm-foreach/ @AlexeySachkov  @Fznamznon
```